tools/amd_build/build_amd.py

```diff
@@ -201,6 +201,68 @@ def remove_hcc(line: str) -> str:
                     sources.write(line)
             print(f"{hip_platform_file} updated")
 
+
+ck_kernel_launch_file = (
+    "third_party/composable_kernel/include/ck_tile/host/kernel_launch.hpp"
+)
+
+
+def add_make_kernel_pt(line: str) -> str:
```

Comment:
oh wow! This is actually patching a ck header file in the pytorch build file? That doesn't sound a good idea. Can we add this into CK itself?